src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -587,11 +587,13 @@ int32_t AppleCryptoNative_SslSetEnabledCipherSuites(SSLContextRef sslContext, co
 
     if (sizeof(SSLCipherSuite) == sizeof(uint32_t))
     {
+#ifndef TARGET_IOS
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
         // macOS
         return SSLSetEnabledCiphers(sslContext, cipherSuites, (size_t)numCipherSuites);
 #pragma clang diagnostic pop   
+#endif
```

Comment:
I assume this if block is never going to be hit on iOS?  It's just strange that this leaves a path through the function that doesn't have an explicit `return`.

Reply:
Fixed